src/testing/recording.ts

```diff
@@ -20,6 +20,8 @@ interface SetupRecordingInput {
   options?: PollyConfig;
 }
 
+const SENSITIVE_HEADER_NAMES = ['authorization'].map((i) => i.toLowerCase());
```

Comment:
Perhaps there's no need to normalize the constant `SENSITIVE_HEADER_NAMES` at all, since we'll only put lowercase values in it?
```suggestion
const SENSITIVE_HEADER_NAMES = ['authorization'];
```

Reply:
Probably not, but I added it incase someone screws up.